compiler/core/lam.ml

```diff
@@ -298,8 +298,8 @@ let rec apply ?(ap_transformed_jsx = false) fn args (ap_info : ap_info) : t =
           Lprim
             {
               primitive =
-                ( Pundefined_to_opt | Pnull_to_opt | Pnull_undefined_to_opt
```

Comment:
Removing the primitive to convert undefined to option makes sense, as the undefined type is de-emphasized (no `Stdlib_undefined` module), so it would be weird to special case this conversion in the compiler.